specification/cognitiveservices/ContentSafetyStudio/main.tsp

```diff
@@ -0,0 +1,429 @@
+  latency: int32;
+}
+
+@doc("Response of latencies.")
+@pagedResult
+model LatencyList {
+  @doc("List of latency.")
+  @items
+  value: LatencyPerTimestamp[];
+
+  @doc("Link to fetch more.")
+  @nextLink
+  nextLink?: string;
+}
+
+@doc("Popular term and it's API count.")
+@resource("topTerms")
+model TopTermCount {
+  #suppress "@azure-tools/typespec-azure-core/key-visibility-required" "Properties in response-only schemas should not be marked readOnly"
+  @key("term")
+  @doc("The term content.")
+  term: string;
+
+  @doc("The API call count of the term.")
+  count: int32;
+}
+
+
+@doc("Response of listing hit terms.")
+@pagedResult
+model TermList {
+  @doc("List of terms.")
+  @items
+  value: TopTermCount[];
+
+  @doc("Link to fetch more terms.")
+  @nextLink
+  nextLink?: string;
+}
+
+@doc("Blocklist hit call counts at a timestamp")
+@resource("hitCalls")
+model BlockListHitCallsPerTimestamp {
+  #suppress "@azure-tools/typespec-azure-core/key-visibility-required" "Properties in response-only schemas should not be marked readOnly"
+  @key("dateTime")
+  @doc("The date time.")
+  dateTime: utcDateTime;
+
+  @doc("The call count of the hit API.")
+  count: int32;
+}
+
+@doc("The detection category.")
+@resource("categories")
+model Category {
+  #suppress "@azure-tools/typespec-azure-core/key-visibility-required" "Properties in response-only schemas should not be marked readOnly"
+  @key("category")
+  @doc("The category of the API. The value could be Hate, Sexual, Violence, Self-Harm")
+  category: CategoryEnum;
+}
+
+@doc("Severity and count the API detected.")
+@resource("severities")
+@parentResource(Category)
+model Severity {
+  #suppress "@azure-tools/typespec-azure-core/key-visibility-required" "Properties in response-only schemas should not be marked readOnly"
+  @doc("The severity the API returned.")
+  @key("severity")
+  severity: SeverityEnum;
+
+  @doc("The count of the severity.")
+  count: int32;
+}
+
+@doc("Response of severities.")
+@pagedResult
+model SeverityList {
+  @doc("List of terms.")
+  @items
+  value: Severity[];
+
+  @doc("Link to fetch more terms.")
+  @nextLink
+  nextLink?: string;
+}
+
+
+@doc("API request count at a timestamp")
+@resource("requestCounts")
+@parentResource(Severity)
+model RequestCountsPerTimestamp {
+  #suppress "@azure-tools/typespec-azure-core/key-visibility-required" "Properties in response-only schemas should not be marked readOnly"
+  @key("dateTime")
+  @doc("The date time.")
+  dateTime: utcDateTime;
+
+  @doc("The count of the request.")
+  count: int32;
+}
+
+
+@doc("Response of listing hit calls.")
+@pagedResult
+model HitCallsList {
+  @doc("List of hit calls.")
+  @items
+  value: BlockListHitCallsPerTimestamp[];
+
+  @doc("Link to fetch more calls.")
+  @nextLink
+  nextLink?: string;
+}
+
+@doc("Error response")
+@error
+model Error {
+  @header("x-ms-error-code")
+  @doc("Error code.")
+  code: int32;
+
+  @doc("Error message.")
+  message: string;
+}
```

Comment:
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-error-response-body-structure

Reply:
Thanks for feedback， updated